packages/components/src/ui/ariakit-tooltip/index.tsx

```diff
@@ -7,12 +7,15 @@ import { Tooltip, TooltipAnchor, useTooltipState } from 'ariakit/tooltip';
  * WordPress dependencies
  */
 import { Children } from '@wordpress/element';
+import deprecated from '@wordpress/deprecated';
 
 /**
  * Internal dependencies
  */
 import Shortcut from '../../shortcut';
+import { TOOLTIP_DELAY } from '../../tooltip/';
 import type { ToolTipProps } from './types';
+import { positionToPlacement as __experimentalPopoverLegacyPositionToPlacement } from '../../popover/utils';
```

Comment:
There's no need to rename the function to `__experimentalPopoverLegacyPositionToPlacement`. The rename is necessary only when exporting the function outside of the components package